src/entities/SearchFormWrapper/SearchFormWrapper.jsx

```diff
@@ -1,14 +1,25 @@
-import { useOutletContext } from "react-router";
+import { useOutletContext, useNavigate } from "react-router";
 import { CustomButton } from "../../shared/ui/CustomButton/CustomButton";
 import { SearchForm } from "../SearchForm/SearchForm";
+import { Outlet } from "react-router-dom";
 
 export const SearchFormWrapper = () => {
   const { searchParam, handleSearch } = useOutletContext();
+  const navigate = useNavigate();
+
+  const handleAddMovieCLick = () => {
+    navigate("/new");
+  };
 
   return (
     <>
+      <Outlet />
       <div className="add-movie-part">
-        <CustomButton className="outlined-button" label="+ADD MOVIE" />
+        <CustomButton
```

Comment:
looks like it should be a `Link` tag from react-router

Reply:
done